demo-shell/src/app/components/cloud/task-details-cloud-demo.component.ts

```diff
@@ -54,6 +58,14 @@ export class TaskDetailsCloudDemoComponent {
         this.goBack();
     }
 
+    onTaskClaimed() {
+        this.taskHeader.ngOnInit();
```

Comment:
If all you have to all is a single method, you can get rid of these methods and call that method directly from the html template. 

@eromano the only thing I have a feeling it's not very correct when one component calls Angular lifecycle methods of another component (there can be anything unexpected inside that OnInit). Maybe having a special public method for that? Like "taskHeader.reload()" ?

Reply:
maybe is better yes